guides/common/modules/proc_puppet-setting-out-of-sync-time-for-puppet-managed-hosts.adoc

```diff
@@ -0,0 +1,54 @@
+[id="puppet_guide_setting_out_of_sync_time_for_puppet_managed_hosts_{context}"]
+= Setting Out of Sync Time for Puppet Managed Hosts
+
+{Project} considers hosts managed by Puppet to be out of sync if the last Puppet report is older than the combined values of `outofsync_interval` and `puppet_interval` set in minutes.
+By default, the Puppet agent on managed hosts runs every 30 minutes.
+The `puppet_interval` is set to 35 minutes and the global `outofsync_interval` is set to 30 minutes.
+
+The effective time after which hosts are considered out of sync is the sum of `outofsync_interval` and `puppet_interval`.
+Setting the global `outofsync_interval` to 30 and the `puppet_interval` to 60 results in a total of 90 minutes after which the host status changes to `out of sync`.
+
+[id="puppet_guide_setting_the_puppet_agent_run_interval{context}"]
+== Setting the Puppet Agent Run Interval
+
+Set the interval when the Puppet Agent runs and sends reports to {Project}.
+
+.Procedure
+. Connect to your managed host using SSH.
+. Add the Puppet Agent run interval to `/etc/puppetlabs/puppet/puppet.conf`, for example `runinterval = 1h`.
+
+[id="puppet_guide_setting_the_puppet_interval_{context}"]
+== Setting the Puppet Interval
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Administer > Settings*.
+. On the *Puppet* tab, edit *Puppet interval*.
```

Comment:
```suggestion
. In the *Puppet interval* field, edit the value to the duration, in minutes, after which hosts reporting using Puppet are considered to be out of sync.
```
Is Puppet interval a field?

Reply:
yes, it's literally `Puppet interval`.

Applied.